test_sort_function.py

```
@@ -0,0 +1,74 @@
+import zipfile
+from pathlib import Path
+
+from sort_function import choose_cover
+
+ROOT_DIR = Path("D:\\comic_library\\comic_examples")
+
+EXPECTED_COVERS = {
+    "Booster Gold Vol. 1 #04 (May 1986)": "Booster Gold (1986-1988) 004-000.jpg",
+    "Green Lantern Emerald Dawn #01 (December 1989)": "Green Lantern - Emerald Dawn (1989-1990) 001-000.jpg",
+    "Runaways by Rowell TPB #02 (October 2018)": "Runaways by Rainbow Rowell & Kris Anka v02 - Best Friends Forever-000.jpg",
+    "Wonder Woman Dead Earth TPB (December 2020)": "Wonder Woman - Dead Earth-000.jpg"
```

Ruff (0.11.9)

Trailing comma missing

Add trailing comma

(COM812)